blueprint/index.md

```diff
@@ -0,0 +1,69 @@
+---
```

Comment:
Using the screenpop image for the header image for this blueprint would probably work well. Specify `image: images/screenpop.png` in the frontmatter if you'd like to do that.